mu4/userscores/qml/MuseScore/UserScores/internal/ScoreItem.qml

```diff
@@ -80,21 +142,26 @@ Item {
         Rectangle {
             anchors.fill: parent
 
-            color: "#FFFFFF"
+            color: "white"
 
             StyledIconLabel {
                 anchors.centerIn: parent
 
                 iconCode: IconCode.PLUS
 
                 font.pixelSize: 50
+                color: "black"
```

Comment:
#000000

Reply:
And why do you prefer hex?
Aren't color names just clearer?

Reply:
It's a bad practice to mix both approaches. And since the "default color names" approach is not as flexible as "HEX" approach it's better to stick with the last one

Reply:
I prefer readability, so symbolic names

Reply:
"back" can be not black (dark grey for example) in qml, defined in styles. If we need black - we should write it explicitly.

Reply:
`black` is the same as `#000000` in any case. The former just more readable